config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-  resources :topics
+  resources :topics, only: [:create, :index]
```

Comment:
I think you also have `show`, `update` and `destroy`.

```suggestion
  resources :topics, only: [:create, :destroy, :index, :show, :update]
```

Reply:
hmm I dont think so, the users should be able to create or see al topics, but not update and destroy existing ones, right? because topics doesnt belong to users, are shared system wide